tests/TestCase/Database/ValueBinderTest.php

```diff
@@ -39,7 +39,7 @@ public function testBind()
         $expected = [
             ':c0' => [
                 'value' => 'value0',
-                'type' => 'string',
+                'type' => null,
```

Comment:
shouldn't there always be a type here?

Reply:
See above:

> I chose to harmonize the default type to null instead of 'string' as PDOStatment::bindValue() ultimately converts null to 'string' before converting string types to its PDO::PARAM_* constants.